wpm/game.py

```diff
@@ -234,14 +234,14 @@ def handle_key(self, key):
         # Browse mode
         if self.start is None or (self.start is not None and
                                   self.stop is not None):
-            if key in (" ", "KEY_LEFT", "KEY_RIGHT"):
+            if key in ("\t", "KEY_LEFT", "KEY_RIGHT"):
                 self.reset(direction=-1 if key == "KEY_LEFT" else 1)
                 return
             elif Screen.is_escape(key):
                 # Exit program
                 raise KeyboardInterrupt()
 
-        if Screen.is_escape(key):
+        if Screen.is_escape(key) or (hard_flag and self.position < len(self.quote.text) and self.quote.text[self.position] != key):
```

Comment:
I have a feeling there must be a much better way. If you look where I update `self.incorrect`, I think you have the logic there already; then you just need to react on your flag.